### PR TITLE
RavenDB-17111 Consider using much shorter thread names to increase visibility

### PR DESCRIPTION
**Pull request by YonatanResh**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-17111/Consider-using-much-shorter-thread-names-to-increase-visibility

### Additional description

we will use shorter thread names in linux, the shorter names probably needs to be refined and documented.

Include details of the change made in this Pull Request or additional notes for the solution. Anything that can be useful for reviewers of this PR.

### Type of change

- Usability

### How risky is the change?

- Low 

### Is it platform specific issue?

- Yes. Linux
- No

### Documentation update

- This change requires a documentation update. Please mark the issue on YouTrack using `Documentation Required` tag.

### Testing 

- It has been verified by manual testing

### CODE REVIEW

File: src/Raven.Server/Documents/Indexes/Index.cs [hunk 1]
```diff
@@ -236,6 +236,7 @@ public bool DeployedOnAllNodes
         internal readonly QueryBuilderFactories _queryBuilderFactories;
 
         private string IndexingThreadName => "Indexing of " + Name + " of " + _indexStorage.DocumentDatabase.Name;
+        private string IndexingShortThreadName => "I:" + IndexNameByFirstLetters() + "-" + _indexStorage.DocumentDatabase.Name;
```
Comment (ayende) [inline src/Raven.Server/Documents/Indexes/Index.cs]: No, that generate something that would be very hard to consume

File: src/Raven.Server/Documents/PeriodicBackup/BackupUploader.cs [hunk 2]
```diff
@@ -259,6 +259,7 @@ private string CombinePathAndKey(string path)
 
             var localUploadStatus = uploadStatus;
             var threadName = $"Upload backup file of database '{_settings.DatabaseName}' to {targetName} (task: '{_settings.TaskName}')";
+            var shortThreadName = $"UBF {_settings.DatabaseName} t {targetName}";
```
Comment (ayende) [inline src/Raven.Server/Documents/PeriodicBackup/BackupUploader.cs]: No way that I'll remember what that means in the future.
In general, we need short, but readable.

Linux thread names are 14 chars, can use a prefix that is unique so we can identify, but generally still must be readable

File: src/Sparrow/LowMemory/LowMemoryNotification.cs [hunk 3]
```diff
@@ -10,7 +10,7 @@ namespace Sparrow.LowMemory
 {
     public class LowMemoryNotification
     {
-        private const string NotificationThreadName = "Low memory notification thread";
+        private readonly string NotificationThreadName = PlatformDetails.RunningOnLinux ? "LMNT" : "Low memory notification thread";
```
Comment (grisha-kotler) [inline src/Sparrow/LowMemory/LowMemoryNotification.cs]: Let's leave the original thread name (there is only one in the system)

File: src/Voron/GlobalFlushingBehavior.cs [hunk 4]
```diff
@@ -4,14 +4,15 @@
...
+using Sparrow.Platform;
 using Sparrow.Utils;
 using Voron.Impl.Journal;
 
 namespace Voron
 {
     public class GlobalFlushingBehavior
     {
-        private const string FlushingThreadName = "Voron Global Flushing Thread";
+        private static readonly string FlushingThreadName = "Voron Flushing Thread";
```
Comment (grisha-kotler) [inline src/Voron/GlobalFlushingBehavior.cs]: const

File: src/Voron/GlobalPrefetchingBehavior.cs [hunk 5]
```diff
@@ -15,7 +16,7 @@ namespace Voron.Impl
 {
     public class GlobalPrefetchingBehavior
     {
-        private const string PrefetchingThreadName = "Voron Global Prefetching Thread";
+        private static readonly string PrefetchingThreadName = "Voron Prefetching Thread";
```
Comment (grisha-kotler) [inline src/Voron/GlobalPrefetchingBehavior.cs]: const

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 6]
```diff
@@ -0,0 +1,412 @@
...
+
+            public Follower(string connection, long term)
+            {
+                _connection = connection;
+                _term = term;
+            }
+
+            public string GetShortName()
+            {
+                return $"Flwr from {_connection}";
```
Comment (grisha-kotler) [inline src/Sparrow.Server/Utils/ThreadNames.cs]:
```suggestion
                return $"Follower {_connection}";
```

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 7]
```diff
@@ -0,0 +1,412 @@
...
+
+            public FollowerAmbassador(string tag, string term)
+            {
+                _tag = tag;
+                _term = term;
+            }
+
+            public string GetShortName()
+            {
+                return $"FlwrAmb for {_tag}";
```
Comment (grisha-kotler) [inline src/Sparrow.Server/Utils/ThreadNames.cs]:
```suggestion
                return $"FollowAmb {_tag}";
```

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 8]
```diff
@@ -0,0 +1,412 @@
...
+
+            public ConsensusLeader(string engineTag, long term)
+            {
+                _engineTag = engineTag;
+                _term = term;
+            }
+
+            public string GetShortName()
+            {
+                return $"CnsLead-{_engineTag} IT {_term}";
```
Comment (grisha-kotler) [inline src/Sparrow.Server/Utils/ThreadNames.cs]:
```suggestion
                return $"ConsensusL-{_engineTag} IT {_term}";
```

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 9]
```diff
@@ -0,0 +1,412 @@
...
+            public HeartbeatsSupervisor(string serverNodeTag, string clusterTag, long term)
+            {
+                _serverNodeTag = serverNodeTag;
+                _clusterTag = clusterTag;
+                _term = term;
+            }
+
+            public string GetShortName()
+            {
+                return $"HBS from {_serverNodeTag} to {_clusterTag}";
```
Comment (grisha-kotler) [inline src/Sparrow.Server/Utils/ThreadNames.cs]:
```suggestion
                return $"HeartbeatS {_serverNodeTag} to {_clusterTag}";
```

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 10]
```diff
@@ -0,0 +1,412 @@
...
+
+            public HeartbeatsWorker(string leader, long term)
+            {
+                _leader = leader;
+                _term = term;
+            }
+
+            public string GetShortName()
+            {
+                return $"HBWCL {_leader} IT {_term}";
```
Comment (grisha-kotler) [inline src/Sparrow.Server/Utils/ThreadNames.cs]:
```suggestion
                return $"HeartbeatW {_leader} IT {_term}";
```

**Review by grisha-kotler — CHANGES_REQUESTED** [review]

File: src/Sparrow/LowMemory/LowMemoryNotification.cs (hunk 3 above)
Comment (YonatanResh) [reply src/Sparrow/LowMemory/LowMemoryNotification.cs]: done.

File: src/Voron/GlobalFlushingBehavior.cs (hunk 4 above)
Comment (YonatanResh) [reply src/Voron/GlobalFlushingBehavior.cs]: done.

File: src/Voron/GlobalPrefetchingBehavior.cs (hunk 5 above)
Comment (YonatanResh) [reply src/Voron/GlobalPrefetchingBehavior.cs]: done.

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 6 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: done.

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 7 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: done.

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 8 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: done.

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 10 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: done.
HBeat

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 9 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: done.
HBeat

**Review by grisha-kotler — APPROVED** [review]

**Comment by arekpalinski** [comment]
Can you please do the rebase, solve the conflicts and check failing tests?

File: src/Raven.Server/Documents/ETL/EtlProcess.cs [hunk 11]
```diff
@@ -628,7 +629,11 @@ public override void Start()
                     if (Logger.IsOperationsEnabled)
                         Logger.Operations($"Failed to run ETL {Name}", e);
                 }
-            }, null, threadName);
+            }, null, new ThreadNames.ThreadInfo
```
Comment (ppekrol) [inline src/Raven.Server/Documents/ETL/EtlProcess.cs]: I would prefer usage like:

```
ThreadNames.ForEtlProcess(parameter1, parameter2);
```

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 12]
```diff
@@ -0,0 +1,412 @@
...
+    }
+    
+    public class ThreadDetails
+    {
+        public interface IThreadDetails
+        {
+            public string GetShortName();
+        }
+
+        public class Elector : IThreadDetails
```
Comment (ppekrol) [inline src/Sparrow.Server/Utils/ThreadNames.cs:210]: probably you can make it private after doing `ThreadNames.ForElector(...)`

File: src/Voron/GlobalPrefetchingBehavior.cs [hunk 13]
```diff
@@ -15,7 +16,7 @@ namespace Voron.Impl
 {
     public class GlobalPrefetchingBehavior
     {
-        private const string PrefetchingThreadName = "Voron Global Prefetching Thread";
+        private const string PrefetchingThreadName = "Voron Prefetching Thread";
```
Comment (ppekrol) [inline src/Voron/GlobalPrefetchingBehavior.cs:19]: better to remove 'Thread' instead?

File: src/Voron/GlobalFlushingBehavior.cs [hunk 14]
```diff
@@ -4,14 +4,15 @@
...
+using Sparrow.Platform;
 using Sparrow.Utils;
 using Voron.Impl.Journal;
 
 namespace Voron
 {
     public class GlobalFlushingBehavior
     {
-        private const string FlushingThreadName = "Voron Global Flushing Thread";
+        private const string FlushingThreadName = "Voron Flushing Thread";
```
Comment (ppekrol) [inline src/Voron/GlobalFlushingBehavior.cs:15]: better to remove 'Thread' instead?

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 15]
```diff
@@ -0,0 +1,412 @@
...
+            private readonly string _destinationUrl;
+            public PullReplicationAsSink(string destinationDatabase, string destinationUrl)
+            {
+                _destinationDatabase = destinationDatabase;
+                _destinationUrl = destinationUrl;
+            }
+
+            public string GetShortName()
+            {
+                return $"RepPS {_destinationDatabase} at {_destinationUrl}";
```
Comment (ppekrol) [inline src/Sparrow.Server/Utils/ThreadNames.cs]: `PllRepSnk`

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 16]
```diff
@@ -0,0 +1,412 @@
...
+
+            public Backup(string backupName, string databaseName)
+            {
+                _backupName = backupName;
+                _databaseName = databaseName;
+            }
+
+            public string GetShortName()
+            {
+                return $"Backup {_databaseName}";
```
Comment (ppekrol) [inline src/Sparrow.Server/Utils/ThreadNames.cs]: `Bkp` ?

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 17]
```diff
@@ -0,0 +1,412 @@
...
+            private readonly long _term;
+
+            public ClusterObserver(long term)
+            {
+                _term = term;
+            }
+
+            public string GetShortName()
+            {
+                return $"Observer {_term}";
```
Comment (ppekrol) [inline src/Sparrow.Server/Utils/ThreadNames.cs:500]: `Obsrvr` ?

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 18]
```diff
@@ -0,0 +1,412 @@
...
+
+            public HeartbeatsWorker(string leader, long term)
+            {
+                _leader = leader;
+                _term = term;
+            }
+
+            public string GetShortName()
+            {
+                return $"HBbeatW {_leader} IT {_term}";
```
Comment (ppekrol) [inline src/Sparrow.Server/Utils/ThreadNames.cs]: `HrtbtWrkr` 

What is `IT` ?

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 19]
```diff
@@ -0,0 +1,412 @@
...
+
+            public ConsensusLeader(string engineTag, long term)
+            {
+                _engineTag = engineTag;
+                _term = term;
+            }
+
+            public string GetShortName()
+            {
+                return $"ConsensusL-{_engineTag} IT {_term}";
```
Comment (ppekrol) [inline src/Sparrow.Server/Utils/ThreadNames.cs]: `CnsnsLdr` ?

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 20]
```diff
@@ -0,0 +1,412 @@
...
+
+            public FollowerAmbassador(string tag, string term)
+            {
+                _tag = tag;
+                _term = term;
+            }
+
+            public string GetShortName()
+            {
+                return $"FollowAmb {_tag}";
```
Comment (ppekrol) [inline src/Sparrow.Server/Utils/ThreadNames.cs]: `FllwrAmb` ?

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 21]
```diff
@@ -0,0 +1,412 @@
...
+
+            public Follower(string connection, long term)
+            {
+                _connection = connection;
+                _term = term;
+            }
+
+            public string GetShortName()
+            {
+                return $"Follower {_connection}";
```
Comment (ppekrol) [inline src/Sparrow.Server/Utils/ThreadNames.cs]: `Fllwr` ?

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 22]
```diff
@@ -0,0 +1,412 @@
...
+
+            public CandidateAmbassador(string engineTag, string tag)
+            {
+                _engineTag = engineTag;
+                _tag = tag;
+            }
+
+            public string GetShortName()
+            {
+                return $"CndAm for {_engineTag}>{_tag}";
```
Comment (ppekrol) [inline src/Sparrow.Server/Utils/ThreadNames.cs:398]: `CnddteAmb` ?

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 23]
```diff
@@ -0,0 +1,412 @@
...
+            private readonly string _engineTag;
+
+            public Candidate(string engineTag)
+            {
+                _engineTag = engineTag;
+            }
+
+            public string GetShortName()
+            {
+                return $"Candidate for {_engineTag}";
```
Comment (ppekrol) [inline src/Sparrow.Server/Utils/ThreadNames.cs]: `Cnddte`

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 24]
```diff
@@ -0,0 +1,412 @@
...
+            private readonly string _name;
+
+            public TransactionMerging(string name)
+            {
+                _name = name;
+            }
+
+            public string GetShortName()
+            {
+                return $"TXMRG {_name}";
```
Comment (ppekrol) [inline src/Sparrow.Server/Utils/ThreadNames.cs]: `TxMrgr`

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 25]
```diff
@@ -0,0 +1,412 @@
...
+            }
+
+            public string GetShortName()
+            {
+                if (_pullReplicationAsHub)
+                {
+                    return $"RepPAH {_databaseName} to {_destination}";
+                }
+
+                return $"RepO {_databaseName} to {_destination}";
```
Comment (ppekrol) [inline src/Sparrow.Server/Utils/ThreadNames.cs]: `OutRpl`

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 26]
```diff
@@ -0,0 +1,412 @@
...
+                _databaseName = databaseName;
+                _destination = destination;
+                _pullReplicationAsHub = pullReplicationAsHub;
+            }
+
+            public string GetShortName()
+            {
+                if (_pullReplicationAsHub)
+                {
+                    return $"RepPAH {_databaseName} to {_destination}";
```
Comment (ppekrol) [inline src/Sparrow.Server/Utils/ThreadNames.cs]: `PllRepHb`

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 27]
```diff
@@ -0,0 +1,412 @@
...
+
+            public IncomingReplication(string dbName, string sourceDbName)
+            {
+                _dbName = dbName;
+                _sourceDbName = sourceDbName;
+            }
+
+            public string GetShortName()
+            {
+                return $"RepInc {_dbName} from {_sourceDbName}";
```
Comment (ppekrol) [inline src/Sparrow.Server/Utils/ThreadNames.cs]: `IncRep`

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 28]
```diff
@@ -0,0 +1,412 @@
...
+
+            public BackupTask(string dbName, string taskName)
+            {
+                _dbName = dbName;
+                _taskName = taskName;
+            }
+
+            public string GetShortName()
+            {
+                return $"Backup {_dbName}";
```
Comment (ppekrol) [inline src/Sparrow.Server/Utils/ThreadNames.cs]: `Bkp` - there is another one with this shortcut?

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 16 above)
Comment (ppekrol) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: We have a duplicate `BackupTask` ?

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 29]
```diff
@@ -0,0 +1,412 @@
...
+            public DeleteBackupFile(string dbName, string targetName, string taskName)
+            {
+                _dbName = dbName;
+                _targetName = targetName;
+                _taskName = taskName;
+            }
+
+            public string GetShortName()
+            {
+                return $"DBF {_dbName} from {_targetName}";
```
Comment (ppekrol) [inline src/Sparrow.Server/Utils/ThreadNames.cs]: `DelBkpFl`

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 30]
```diff
@@ -0,0 +1,412 @@
...
+
+            public Index(string idxName, string dbName)
+            {
+                _idxName = idxName;
+                _dbName = dbName;
+            }
+
+            public string GetShortName()
+            {
+                return $"IX:{_dbName} {_idxName}";
```
Comment (ppekrol) [inline src/Sparrow.Server/Utils/ThreadNames.cs]: `Idx`

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 31]
```diff
@@ -0,0 +1,412 @@
...
+            public UploadBackupFile(string dbName, string targetName, string taskName)
+            {
+                _dbName = dbName;
+                _targetName = targetName;
+                _taskName = taskName;
+            }
+
+            public string GetShortName()
+            {
+                return $"UBF {_dbName} to {_targetName}";
```
Comment (ppekrol) [inline src/Sparrow.Server/Utils/ThreadNames.cs]: `UplBkpFl` ?

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 32]
```diff
@@ -0,0 +1,412 @@
...
+
+            public EtlProcess(string tag, string name)
+            {
+                _tag = tag;
+                _name = name;
+            }
+
+            public string GetShortName()
+            {
+                return $"{_tag} {_name}";
```
Comment (ppekrol) [inline src/Sparrow.Server/Utils/ThreadNames.cs]: `Etl ` ?

File: src/Sparrow.Server/Utils/ThreadNames.cs [hunk 33]
```diff
@@ -0,0 +1,412 @@
...
+            private readonly string _source;
+
+            public Elector(string source)
+            {
+                _source = source;
+            }
+
+            public string GetShortName()
+            {
+                return $"Elector {_source}";
```
Comment (ppekrol) [inline src/Sparrow.Server/Utils/ThreadNames.cs]: `Elctr` ?

File: src/Voron/GlobalPrefetchingBehavior.cs (hunk 13 above)
Comment (YonatanResh) [reply src/Voron/GlobalPrefetchingBehavior.cs:19]: 15 chars will give us just "Voron Prefetchi"

File: src/Voron/GlobalFlushingBehavior.cs (hunk 14 above)
Comment (YonatanResh) [reply src/Voron/GlobalFlushingBehavior.cs:15]: 15 chars will give us just "Voron Flushing"

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 17 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs:500]: there is no need to shorten thread names that are less then 15 chars unless tern reach numbers that takes more than 6 chars

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 18 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: IT = in term

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 33 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: done

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 32 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: done

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 30 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: done

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 27 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: done

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 26 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: done

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 25 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: done

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 24 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: done

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 23 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: done

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 22 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs:398]: to long

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 21 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: done

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 20 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: done

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 19 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: done

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 18 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: done

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 15 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: done

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 31 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: to long?

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 29 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: to long?

File: src/Raven.Server/Documents/ETL/EtlProcess.cs (hunk 11 above)
Comment (YonatanResh) [reply src/Raven.Server/Documents/ETL/EtlProcess.cs]: done

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 12 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs:210]: could only make it internal, grisha thought we should just leave it public

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 16 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: done
this one time backup.
i changed the other one to BkpTsk (Periodic Backup)

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 28 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: done
BkpTsk

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 31 above)
Comment (ppekrol) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: At least `UpBkp` or sth like that `UBF` is to vague

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 29 above)
Comment (ppekrol) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: At least `DlBkp` or sth like that `DBF` is to vague

Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: done

File: src/Sparrow.Server/Utils/ThreadNames.cs (hunk 31 above)
Comment (YonatanResh) [reply src/Sparrow.Server/Utils/ThreadNames.cs]: done

**Pull request merged**